node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -45,12 +45,14 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
 
   val peer: Peer
 
-  protected val initialSyncDone: Option[Promise[Done]]
```

Comment:
So this is totally gone now? How do we know when IBD is done for a specific node ?

Reply:
We never used it anywhere